airbyte-integrations/connectors/source-oracle/src/test/java/io/airbyte/integrations/source/oracle/OracleJdbcStandardSourceTest.java

```diff
@@ -31,11 +31,11 @@
 import io.airbyte.integrations.source.jdbc.AbstractJdbcSource;
 import io.airbyte.integrations.source.jdbc.test.JdbcSourceStandardTest;
 import java.math.BigDecimal;
+import java.sql.*;
```

Comment:
we tend to avoid using the `.*` syntax. there's a setting in intellij that lets you set it very high so it doesn't kick in.

![Screen Shot 2021-03-30 at 4 20 32 PM](https://user-images.githubusercontent.com/9092207/113068786-df62e900-9173-11eb-86b3-54077ce8ac8c.png)

Reply:
i'm not sure it does. it's an added style decision we made on top of the google java style